hat/assets/js/apps/Iaso/domains/dataSources/components/DataSourceDialogComponent.js

```diff
@@ -152,6 +158,14 @@ export const DataSourceDialogComponent = ({
         value: v.id,
     }));
 
+    const urlErrors = translateErrors(
+        form.credentials_dhis2_url,
+        formatMessage,
+    );
+    const userPasswordErrors = translateErrors(
+        form.credentials_dhis2_password,
+        formatMessage,
+    );
```

Comment:
Not blocking: you could make a hook `useTranslatedDhis2Errors` and avoid passing `formatMessage `to your function `translateErrors`